ipynb/primary-processing/05-alpha_diversity.md

```diff
@@ -57,20 +57,16 @@ Once we have the rarefactions, we can then compute the diversity of every sample
 
 ```python
 >>> for table, depth, rarefaction, multi_directory, adiv_directory, keybase in parameter_iterator():
-...     !parallel_alpha_diversity.py -i $multi_directory \
-...                                  -o $adiv_directory \
-...                                  -t $greengenes_tree \
-...                                  -m PD_whole_tree,chao1,observed_otus,shannon \
-...                                  -O $cpu_count
+...     !alpha_diversity.py -i $multi_directory \
+...                         -o $adiv_directory \
+...                         -t $greengenes_tree \
+...                         -m PD_whole_tree,chao1,observed_otus,shannon
 ```
 
 We're going to aggregate the diversity calculations from the multiple rarefactions.
 
 ```python
 >>> for table, depth, rarefaction, multi_directory, adiv_directory, keybase in parameter_iterator():
-...     # sometimes QIIME leaves this directory, unsure why.
```

Comment:
i've also seen the directory left when it doesn't silently fail. it's
possible that the filesystems lag too much on barnacle

On Fri, Mar 11, 2016 at 2:01 PM, Jose Navas <notifications@github.com>
wrote:

> In ipynb/primary-processing/05-alpha_diversity.md
> <https://github.com/biocore/American-Gut/pull/205#discussion_r55895354>:
>
> >  ```
> >
> >  We're going to aggregate the diversity calculations from the multiple rarefactions.
> >
> >  ```python
> >  >>> for table, depth, rarefaction, multi_directory, adiv_directory, keybase in parameter_iterator():
> > -...     # sometimes QIIME leaves this directory, unsure why.
>
> For future reference - the times that I've seen QIIME leaving this
> directory is when the parallel call fails silently - so this may be
> suggesting that parallel alpha diversity was already failing sometimes and
> this piece of code was preventing detecting it in some other way.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/American-Gut/pull/205/files#r55895354>.
>